website/docs/docker.md

```diff
@@ -175,9 +175,9 @@ services:
     ports:
       - '4873:4873'
     volumes:
-      - './storage:/verdaccio/storage'
-      - './config:/verdaccio/conf'
-      - './plugins:/verdaccio/plugins'
+      - './storage:/storage'
```

Comment:
The volumes start with `/verdccio` how this works for you? https://github.com/verdaccio/verdaccio/blob/master/Dockerfile#L38 I don't get it.

Reply:
I will try some debug to find why